lib/ancestry/instance_methods.rb

```diff
@@ -336,13 +348,13 @@ def unscoped_descendants
     # works with after save context (hence before_last_save)
     def unscoped_current_and_previous_ancestors
       unscoped_where do |scope|
-        scope.where id: (ancestor_ids + ancestor_ids_before_last_save).uniq
+        scope.where self.ancestry_base_class.ancestry_id_column => (ancestor_ids + ancestor_ids_before_last_save).uniq
       end
     end
 
     def unscoped_find id
       unscoped_where do |scope|
-        scope.find id
```

Comment:
`find id` will throw an error if id doesn't exist. but `find_by` will return `nil`. do we want it to fail if `find_by` return nil?